script/provider/provider.lua

```diff
@@ -706,20 +702,12 @@ proto.on('textDocument/semanticTokens/range', function (params)
     workspace.awaitReady()
     local _ <close> = progress.create(lang.script.WINDOW_PROCESSING_SEMANTIC_RANGE, 0.5)
     local core = require 'core.semantic-tokens'
-    local cache  = files.getOpenedCache(uri)
-    local start, finish
-    if cache and not cache['firstSemantic'] then
-        cache['firstSemantic'] = true
-        start  = 0
-        finish = #files.getText(uri)
-    else
-        start, finish = converter.unpackRange(uri, params.range)
-    end
```

Comment:
After the server registers `range`, VSCode no longer sends `full` requests. This will cause the coloring on the minimap to be incomplete when the file is opened for the first time, so I take the initiative to change the first coloring to full

Reply:
Ah, I actual took that part out to try and get something working and forgot to put it back. I'll revert that in a little bit :)